Gemfile.lock

```diff
@@ -785,6 +786,3 @@ DEPENDENCIES
   validate_url (= 1.0.0)
   vcr
   webmock
-
-BUNDLED WITH
-   1.10.5
```

Comment:
I think this was a problem with bundle update. What exactly the gem that you want update?